astroquery/alma/tests/test_alma_remote.py

```diff
@@ -29,27 +28,23 @@
                 'QA2 Status', 'Group ous id', 'Pub'}
 
 
-def get_client():
+@pytest.fixture
+def alma(request):
+    """
+    Returns an alma client class. `--alma-site` pytest option can be used
+    to have the client run against a specific site
+    :param request: pytest request fixture
```

Comment:
doesn't matter here as we don't docstring test functions, but normally we use numpydoc style docstring formatting